docs/semgrep-assistant/getting-started.md

```diff
@@ -119,6 +119,32 @@ Semgrep Assistant messages only appear in your PR comments for rules that are se
   ![ Policies modes](/img/semgrep-assistant-comment.png)
 * You have selected PR/MR comments in **Semgrep AppSec Platform > Settings > Deployment** in the **Code** section.
 
+### Add Memories (beta)
+
+Assistant Memories allows users to tailor Assistant's remediation guidance to their organization's standards and defaults on a per-project, per-rule basis. Whenever Assistant gives a suggested fix,  users can provide feedback by adding custom instructions.
```

Comment:
```suggestion
Assistant Memories allow users to tailor Assistant's remediation guidance to their organization's standards and defaults on a per-project, per-rule basis. Whenever Assistant gives a suggested fix,  users can provide feedback by adding custom instructions.
```
Not sure if single or plural here

Reply:
Is it possible to be more specific than “users” about who can make these changes? Might it be Semgrep admins or more generally, AppSec teams? When I read this I wonder if a developer can add custom instructions.

Reply:
Basically, anyone who can see the suggested fix can create a memory, so it's not just admin. I'll change this to "AppSec teams and developers"